pinot-core/src/main/java/org/apache/pinot/core/operator/docidsets/AndDocIdSet.java

```diff
@@ -107,7 +107,7 @@ public BlockDocIdIterator iterator() {
 
     // evaluate the bitmaps in the order of the lowest matching num docIds comes first, so that we minimize the number
     // of containers (range) for comparison from the beginning, as will minimize the effort of bitmap AND application
-    bitmapBasedDocIdIterators.sort(Comparator.comparing(x -> x.getDocIds().getCardinality()));
+    bitmapBasedDocIdIterators.sort(Comparator.comparing(x -> x.getDocIds().getContainerCount()));
```

Comment:
can you also update the comment just above?

Reply:
Will make that change - Looking for a way to becnhmark this change. Can you recommend something ? Do we have a common tool to do that ?